configuration/defaults/staging/configuration.yaml

```diff
@@ -0,0 +1,269 @@
+##########################################################
+###############          Staging           ###############
+############### Cardano Node Configuration ###############
+##########################################################
+
+
+##### Locations #####
+
+GenesisFile: genesis.json
+SocketPath: db/node.socket
+
+
+##### Core protocol parameters #####
+
+# This is the instance of the Ouroboros family that we are running.
+# The node also supports various test and mock instances.
+# "RealPBFT" is the real (ie not mock) (permissive) OBFT protocol, which
+# is what we use on mainnet in Byron era.
+Protocol = "RealPBFT";
```

Comment:
This is not YAML format ;)

Reply:
this should be fixed by the latest commit